src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { PunsList } from "./components/PunsList/PunsList";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header className="header">
+        <h1 className="header__title">Trocadilho Awards</h1>
+        <nav className="header__nav">
+          <p>LIST OF PUNS</p>
+          <p>ADD PUN</p>
+          <p>RANKING</p>
+        </nav>
       </header>
+      <PunsList />
     </div>
```

Comment:
We can transfer this to a separate component and just render one component here instead of this whole HTML, what do you think?

Reply:
Yes, I thought about it! I decided to do like this for now just to check the general style, but I agree it's better as a component